packages/terra-arrange/tests/nightwatch/ArrangeTests.jsx

```diff
@@ -12,6 +12,7 @@ const BaseTests = () => (
       <li><Link to="/tests/arrange-tests/aligned-individually-fit-start">Individually Aligned FitStart</Link></li>
       <li><Link to="/tests/arrange-tests/aligned-individually-fill">Individually Aligned Fill</Link></li>
       <li><Link to="/tests/arrange-tests/aligned-individually-fit-end">Individually Aligned FitEnd</Link></li>
+      <li><Link to="/tests/arrange-tests/aligned-all-arrange-long-word">Aligned Array with LongWords</Link></li>
```

Comment:
The `to` prop in the Link component should match the Route component's `path` prop. Need to add an `s` after `long-word`